sqlmesh/magics.py

```diff
@@ -791,6 +791,14 @@ def rollback(self, context: Context, line: str) -> None:
         """Rollback SQLMesh to the previous migration."""
         context.rollback()
 
+    @magic_arguments()
+    @line_magic
+    @pass_sqlmesh_context
+    def clean(self, context: Context, line: str) -> None:
+        """Clears the SQLMesh cache and any build artifacts."""
+        context.clear_caches()
+        context.console.log_success("SQLMesh cache and build artifacts cleared")
```

Comment:
Do we like this feedback here? Some methods have it, some don't.

Reply:
Yeah, here it probably makes more sense, since there's no exit code to rely on like in case of the CLI.